### ISSUE TITLE
PGPR-master\train_agent.py:47: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.

### ISSUE THREAD

**Issue by Pursuerwener** [orcax/PGPR#18]
if i run this command python test_agent.py --dataset beauty --run_path True
i always get this warning

PGPR-master\train_agent.py:47: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead. (Triggered internally at  C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten
\src\ATen\native\cuda\Indexing.cu:967.)
  actor_logits[1 - act_mask] = -999999.0

then it will run again and run normally. i can get results, but it affects my debugging. Can you tell me how to solve the warning.
please, Thanks

**Comment by HolyGod33** [comment]
it's torch version problem, because in code, auther use "act_logit[1 - act_mask]" to mask the useless action.  Now you can use "act_logit[~act_mask]"
